doc/ledger3.texi

```diff
@@ -6117,8 +6124,9 @@ Set the width in characters of the amount column in the
 Anonymize registry output, mostly for sending in bug reports.
 
 @item --auto-match
-Automatically match accounts from ledger journal for transactions
-during csv convert.
+When generating ledger transaction from CSV file using the
+@command{convert} command, automatically match account from ledger
```

Comment:
Minor corrections: `automatically match [the] account from [the] [L]edger journal.`